### ISSUE TITLE
Unskip Flow Sharing test incompatible with A11y in 6.6.x

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#8077]
Flow Sharing in 6.6.x are incompatible with A11y. This is probably due to a missing backport PR since they’re passing in the trunk. 
They skipped in this [PR|https://github.com/shopware/SwagCommercial/pull/239/files].

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40803**



[created from NEXT-40803, comment 548750]

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40803**

 Jira Issue URL: [https://shopware.atlassian.net/browse/NEXT-40803](https://shopware.atlassian.net/browse/NEXT-40803)



[created from NEXT-40803, comment 548751]
